community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/setup_network_storage.py

```diff
@@ -30,6 +31,8 @@
 from more_executors import Executors, ExceptionRetryPolicy
 
 
+log = logging.getLogger()
```

Comment:
for this root logger, what would be destination file ?

Reply:
It depends on what "main" uses it.